CMakeLists.txt

```diff
@@ -98,24 +98,13 @@ message(STATUS "GIT_SHA1 ${GIT_SHA1}")
 set(SYSCONFDIR ${SYSCONF_INSTALL_DIR})
 set(SHAREDIR ${CMAKE_INSTALL_DATADIR})
 
-#####
-## handle BUILD_OWNCLOUD_OSX_BUNDLE
-# BUILD_OWNCLOUD_OSX_BUNDLE was not initialized OR set to true on OSX
-if(APPLE AND (NOT DEFINED BUILD_OWNCLOUD_OSX_BUNDLE OR BUILD_OWNCLOUD_OSX_BUNDLE))
-    set(BUILD_OWNCLOUD_OSX_BUNDLE ON)
+# Build MacOS app bundle if wished
+if(APPLE AND BUILD_OWNCLOUD_OSX_BUNDLE)
```

Comment:
our release builds are made already with `BUILD_OWNCLOUD_OSX_BUNDLE` defined ?

Reply:
No, for that I have opened a pull request at brander https://github.com/nextcloud-gmbh/brander/pull/516

Reply:
@mgallien Pr for the dailies https://github.com/nextcloud/client-building/pull/96